### DISCUSSION TITLE
New webUI - Shift

### DISCUSSION THREAD

**Discussion by SJ50** [haugene/docker-transmission-openvpn#1920] [Ideas]
## Before creating this request I have:

**REQUIRED**
- [x] Read through the pinned issues for related problems
- [x] Searched for similar [provider issues][pi] and [container issues][ci]
- [x] [Read the documentation][rd], especially the troubleshooting section and FAQ
- [x] Tried to add as much relevant information to the request as possible
- [x] Agreed that my issue will be closed if I do not follow this template and will remain closed until I complete the template

[pi]: https://github.com/haugene/vpn-configs-contrib/issues
[ci]: https://github.com/haugene/docker-transmission-openvpn/issues
[rd]: https://haugene.github.io/docker-transmission-openvpn/

### Is your feature request related to a problem? Please describe.

**REQUIRED**
```txt
NO
```

### Describe the solution you'd like

**REQUIRED**
```txt
Is it possible to add new well-maintained and minimalist web UI named Shift (https://github.com/killemov/Shift)?
```

### Describe alternatives you've considered

**REQUIRED**
```txt
No alternatives
```
### Additional context

*Optional*
```txt
<placeholder for optional>
```

**Comment by pkishino** [comment]
Thanks for the suggestion.
Before moving forward with this I would like to ask for that repo to produce releases to better manage adding this.. I want to do the same for the other UIs we use to avoid problems moving forward.

**Comment by SJ50** [comment]
I will open issue there asking for release.

**Reply by Prikith** [reply]
i commented this over on shift's repo as well but you can just use a commit id to get a zip, https://github.com/killemov/Shift/archive/25c897e3befcce4e2d4cf46a540f86aa4023dced.zip as an example for the latest commit

**Reply by haugene** [reply]
Instead of using a specific commit you can also target branch name, like https://github.com/killemov/Shift/archive/master.tar.gz would always be the latest commit to master.

**Reply by haugene** [reply]
Updated it here: https://github.com/haugene/docker-transmission-openvpn/commit/8744279e2cd191486ca5ae21e8e051b7b1000b71
Let me know if that for some reason breaks it. But it worked fine locally for me at least.

**Reply by Prikith** [reply]
that was the proposed solution but @pkishino wanted a versioned/tagged release, see https://github.com/killemov/Shift/issues/20

**Reply by haugene** [reply]
Aha. I had not seen that thread. Maybe I was a bit quick. I do agree that it's nice to a versioned release. But there's maintenance in updating those as well.

We should find a common solution here for all UIs. The other UIs are mostly un-versioned as well.

**Reply by pkishino** [reply]
yeah..following other project styles I thought a fixed release and then have a dependencybot auto-creating PR's with updates that we can review would be the nice and avoid stability problems, plus easier to debug as people won't have different versions of various dependencies

**Comment by pkishino** [comment]
please check #1926 and let me know how it goes
